schemas/test/en/test_calculated_summary_dependent_questions.json

```diff
@@ -147,6 +149,412 @@
+                                        }
+                                    ]
+                                }
+                            }
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "repeating-section",
+            "title": "Additional site details",
+            "repeat": {
+                "for_list": "additional_sites_name",
+                "title": {
+                    "text": "{repeat_title_placeholder}",
+                    "placeholders": [
+                        {
+                            "placeholder": "repeat_title_placeholder",
+                            "transforms": [
+                                {
+                                    "arguments": {
+                                        "delimiter": " ",
+                                        "list_to_concatenate": [
+                                            {
+                                                "source": "answers",
+                                                "identifier": "business-name"
+                                            }
+                                        ]
+                                    },
+                                    "transform": "concatenate_list"
+                                }
+                            ]
+                        }
+                    ]
+                }
+            },
+            "summary": {
+                "show_on_completion": false,
+                "page_title": "Details of Additional Sites in Great Britain",
+                "collapsible": false
+            },
+            "show_on_hub": true,
+            "groups": [
+                {
+                    "id": "list-collector-group",
+                    "blocks": [
+                        {
+                            "id": "number-of-employees-working-at-this-additional-site",
+                            "type": "Question",
+                            "question": {
+                                "id": "question-number-of-employees-working-at-this-additional-site",
+                                "title": "In June 2019, what was the number of full-time and part-time employees that your business paid from its payroll, for this site?",
+                                "guidance": {
+                                    "contents": [
+                                        {
+                                            "description": "<strong>Include:</strong>"
+                                        },
+                                        {
+                                            "list": [
+                                                "full-time, over 30 hours a week, and part-time, 30 hours or less, employees aged 16 years and over. This includes individuals <strong>working from home</strong> and those temporarily absent but still employed, for example those on parental leave or sick leave",
```

Comment:
Only minor, but I would consider de-Bresing this schema, as a lot of these descriptions are quite wordy but are not relevant to the test.

Reply:
I removed all guidances and descriptions and shortened questions, renamed group etc. It should be fine now.